web/client/utils/MapInfoUtils.js

```diff
@@ -152,12 +152,16 @@ const MapInfoUtils = {
         const center = {x: lngCorrected, y: props.point.latlng.lat};
         let centerProjected = CoordinatesUtils.reproject(center, 'EPSG:4326', props.map.projection);
         let bounds = MapInfoUtils.getProjectedBBox(centerProjected, resolution, rotation, size, null);
+        let queryLayers = layer.name;
+        if (layer.queryable) {
```

Comment:
It is custom in the sense that the upstream there is not support for WMS sublayer handling AFAICS. Say you have a WMS layer published as

    somelayer
       |--> sublayer1
       |--> sublayer2
       etc

If you pass `somelayer` as layer parameter in the WMS request, it means the main layer including all possible sublayers. You can however select only certain sublayers by passing instead a comma-separated list of sublayer names, i.e. `sublayer1,sublayer2`. QGIS server (and possibly other WMS servers) can publish in the capabilities whether individual sublayers are "queryable" aka identifyable.

As for the name "queryable", I have no problems renaming it to "identifyable" or something else.